tasks/version.js

```diff
@@ -1,8 +1,7 @@
-"use strict";
-
 var exec = require( "child_process" ).exec;
 
 module.exports = function( grunt ) {
+	"use strict";
```

Comment:
This is a Node.js file, those are better served by a global `"use strict"`; pragma. We should have a separate config for Node.js scripts & the browser files, see e.g. how https://github.com/jquery/jquery has that configured.

Reply:
You are right @mgol , my bad. I'm going to fix all task files.

Many thanks

Reply:
Removed all `use strict` from all task files. I have had to add some extra files and configuration in the `Gruntfile` to have a configuration like jQuery project. I hope now the configuration is the correct one.